ViewModels/LeverPostingApplyViewModel.cs

```diff
@@ -59,6 +62,75 @@ public string ToJson
             }
         }
 
+        public MultipartFormDataContent ToDataFrom()
+        {
+            var httpContent = new MultipartFormDataContent
+            {
+                // Double qoutes are required for keys as the Lever API doesn't accept the key without double qoutes.
+                // Solution from: https://stackoverflow.com/a/21874413
+                { new StringContent(!string.IsNullOrEmpty(Comments) ? Comments : string.Empty), @"""comments""" },
```

Comment:
I'd suggest moving the double quotes to it's own method (e.g. `FormatForLever`) as it's being used everywhere.